regression/__snapshots__/validateExamplesRegression_latestOnly.test.ts.snap

```diff
@@ -1506,19 +1506,6 @@ Array [
     "severity": 0,
     "source": "response",
   },
-  Object {
```

Comment:
no sure why this part is removed from the snapshot. Any idea?

Reply:
i think it is because you are bringing an updated version of sway which contains a bug fix from Ray. So it should be fine.